Xero.NetStandard.OAuth2/Model/Accounting/TimeZone.cs

```diff
@@ -685,7 +685,199 @@ public enum TimeZone
+        [EnumMember(Value = "EASTERNSTANDARDTIME(MEXICO)")]
+        EASTERNSTANDARDTIMEMEXICO = 127,
+
+        /// <summary>
+        /// Enum HAITISTANDARDTIME for value: HAITISTANDARDTIME
+        /// </summary>
+        [EnumMember(Value = "HAITISTANDARDTIME")]
+        HAITISTANDARDTIME = 128,
+
+        /// <summary>
+        /// Enum LORDHOWESTANDARDTIME for value: LORDHOWESTANDARDTIME
+        /// </summary>
+        [EnumMember(Value = "LORDHOWESTANDARDTIME")]
+        LORDHOWESTANDARDTIME = 129,
+
+        /// <summary>
+        /// Enum MAGALLANESSTANDARDTIME for value: MAGALLANESSTANDARDTIME
+        /// </summary>
+        [EnumMember(Value = "MAGALLANESSTANDARDTIME")]
+        MAGALLANESSTANDARDTIME = 130,
+
+        /// <summary>
+        /// Enum MARQUESASSTANDARDTIME for value: MARQUESASSTANDARDTIME
+        /// </summary>
+        [EnumMember(Value = "MARQUESASSTANDARDTIME")]
+        MARQUESASSTANDARDTIME = 131,
+
+        /// <summary>
+        /// Enum NORFOLKSTANDARDTIME for value: NORFOLKSTANDARDTIME
+        /// </summary>
+        [EnumMember(Value = "NORFOLKSTANDARDTIME")]
+        NORFOLKSTANDARDTIME = 132,
+
+        /// <summary>
+        /// Enum NORTHKOREASTANDARDTIME for value: NORTHKOREASTANDARDTIME
+        /// </summary>
+        [EnumMember(Value = "NORTHKOREASTANDARDTIME")]
+        NORTHKOREASTANDARDTIME = 133,
+
+        /// <summary>
+        /// Enum SAINTPIERRESTANDARDTIME for value: SAINTPIERRESTANDARDTIME
+        /// </summary>
+        [EnumMember(Value = "SAINTPIERRESTANDARDTIME")]
+        SAINTPIERRESTANDARDTIME = 134,
+
+        /// <summary>
+        /// Enum SAKHALINSTANDARDTIME for value: SAKHALINSTANDARDTIME
+        /// </summary>
+        [EnumMember(Value = "SAKHALINSTANDARDTIME")]
+        SAKHALINSTANDARDTIME = 135,
+
+        /// <summary>
+        /// Enum TOCANTINSSTANDARDTIME for value: TOCANTINSSTANDARDTIME
+        /// </summary>
+        [EnumMember(Value = "TOCANTINSSTANDARDTIME")]
+        TOCANTINSSTANDARDTIME = 136,
+
+        /// <summary>
+        /// Enum TRANSBAIKALSTANDARDTIME for value: TRANSBAIKALSTANDARDTIME
+        /// </summary>
+        [EnumMember(Value = "TRANSBAIKALSTANDARDTIME")]
+        TRANSBAIKALSTANDARDTIME = 137,
+
+        /// <summary>
+        /// Enum TURKSANDCAICOSSTANDARDTIME for value: TURKSANDCAICOSSTANDARDTIME
+        /// </summary>
+        [EnumMember(Value = "TURKSANDCAICOSSTANDARDTIME")]
+        TURKSANDCAICOSSTANDARDTIME = 138,
+
+        /// <summary>
+        /// Enum UTC08 for value: UTC08
+        /// </summary>
+        [EnumMember(Value = "UTC08")]
+        UTC08 = 139,
+
+        /// <summary>
+        /// Enum UTC09 for value: UTC09
+        /// </summary>
+        [EnumMember(Value = "UTC09")]
+        UTC09 = 140,
+
+        /// <summary>
+        /// Enum YUKONSTANDARDTIME for value: YUKONSTANDARDTIME
+        /// </summary>
+        [EnumMember(Value = "YUKONSTANDARDTIME")]
+        YUKONSTANDARDTIME = 141
```

Comment:
Would alphabetizing the enum values in the spec cause issues here? Since the enums are being assigned a numeric value?

Reply:
@rlayhbrian and I had a look into this and it still works as expected even with the values changed